outputs.tf

```diff
@@ -3,6 +3,11 @@ output "aws_logs_bucket" {
   value       = "${aws_s3_bucket.aws_logs.id}"
 }
 
+output "cloudtrail_cloudwatch_logs_arn" {
+  description = "The ARN of the CloudWatch Logs group storing CloudTrail Events."
+  value       = "${aws_cloudwatch_log_group.main.*.arn}"
```

Comment:
Don't you need to do something wonky here to account for when this doesn't exist due to the conditional?

Reply:
the `*` splat handles that bit if it doesn't exist